arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -4,11 +4,55 @@
     "version": "2016-06-27-preview",
     "title": "ContainerRegistry"
   },
-  "host": "https://management.azure.com",
+  "host": "management.azure.com",
   "schemes": [
-    "http"
+    "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/moveResources": {

```

Comment:
In case you still want to keep this API, please add descriptions:
post: "Moves the resource to another resource group or subscription."
resourceGroup: "The name of the resource group within the user's subscription."
moveRequest: ?